### ISSUE TITLE
How to train the  NMT+BPE model?

### ISSUE THREAD

**Issue by yapingzhao** [tensorflow/nmt#314]
Hi,
I use the following command for model training.
python -m nmt.nmt --src=mn --tgt=zh --vocab_prefix=nmt/tmp/nmt_data/vocab --
train_prefix=nmt/tmp/nmt_data/train.mn-zh --dev_prefix=nmt/tmp/nmt_data/dev.mn-zh --
test_prefix=nmt/tmp/nmt_data/test --out_dir=nmt/tmp/nmt_model --num_train_steps=1200 --
steps_per_stats=100 --num_layers=2 --num_units=128 --batch_size=128 --dropout=0.2 --
metrics=bleu  --subword_option=bpe
I would like to ask, parallel bilingual dictionary need to use subword-nmt preprocess?
Looking forward to your advice or answers.
Best regards,

yapingzhao
